### DISCUSSION TITLE
Bad search experience

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#15991] [Other Feature Feedback, Questions, & Ideas] [answered]
I know that `Code Search` is the replacement, I'm using this to give feedback about `Navigation`

Here's a search:
https://github.com/github/docs/search?q=%22The+name+of+the+runner+that+is+running+this+job%22

The first couple of times I load the page, I get this:
![image](https://user-images.githubusercontent.com/2119212/166493488-1b63fe03-e18a-42a8-927d-c4951b2f3c36.png)

Eventually, loading exactly the same url yields this:
![image](https://user-images.githubusercontent.com/2119212/166493800-c7e59cf1-1cd7-4694-ae59-f2ce772b0633.png)

It's really frustrating. If the search fails, I should not get a thing saying "no results", I should get a thing saying "search timed out, try again in a minute" or something.

**Comment by colinwm** [comment, marked as answer]
Sorry about that! We will soon be integrating the new code search experience (cs.github.com) into this page, and we will correctly indicate when a search times out vs. has no results. For now, please try cs.github.com!
